src/main/java/org/sunbird/profile/service/UserBulkUploadService.java

```diff
@@ -141,21 +158,30 @@ private void processBulkUpload(HashMap<String, String> inputDataMap) throws IOEx
                         errorDetails = nextRow.createCell(5);
                     }
                     totalRecordsCount++;
-                    if (errList.isEmpty()) {
-                        boolean isUserCreated = userUtilityService.createUser(userRegistration);
-                        if (isUserCreated) {
-                            noOfSuccessfulRecords++;
-                            statusCell.setCellValue(Constants.SUCCESS.toUpperCase());
-                            errorDetails.setCellValue("");
+                    if (!errList.isEmpty()) {
+                        str.append("Failed to Process the Uploaded File. Error Details - [").append(errList).append("]");
+                        failedRecordsCount++;
+                        statusCell.setCellValue(Constants.FAILED.toUpperCase());
```

Comment:
Can we use different constant which has all upper case instead of calling toUpperCase